onnxruntime/core/providers/coreml/builders/impl/conv_op_builder.cc

```diff
@@ -107,10 +151,26 @@ Status ConvOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const N
     CreateCoreMLWeight(*coreml_conv->mutable_bias(), bias_tensor);
   }
 
-  *layer->mutable_input()->Add() = input_defs[0]->Name();
-  *layer->mutable_output()->Add() = node.OutputDefs()[0]->Name();
+  if (is_1d_conv) {
+    std::string conv_output_name = model_builder.GetUniqueName(MakeString(node.Name(), "_conv_output"));
```

Comment:
>"_conv_output [](http://example.com/codeflow?start=87&length=13)

Nit, be consistent, ""_Conv_output"

Reply:
Ah.. This lowerletter case is for "input/output name" like the `_expandDims` before
and `_Conv_squeeze` and `_Conv_expand` are for layer names from we did in `clip_op_builder.cc`. I believe this is somehow consistent?